icecandidate.go

```diff
@@ -141,3 +141,11 @@ func convertTypeFromICE(t ice.CandidateType) (ICECandidateType, error) {
 		return ICECandidateType(t), fmt.Errorf("unknown ICE candidate type: %s", t)
 	}
 }
+
+func (c ICECandidate) String() string {
+	ic, err := c.toICE()
+	if err != nil {
+		return fmt.Sprintf("%#v", c)
```

Comment:
Is this going to obscure an error. It looks like `toICE` will tell you if the IP is busted (or invalid candidate type) might be useful info.